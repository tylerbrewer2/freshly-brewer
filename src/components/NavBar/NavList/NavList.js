import React from 'react';
import styled from 'styled-components';

const NavListCore = ({ className, children }) => {
  return(
    <ul className={className}>
      {children}
    </ul>
  )
}

const NavList = styled(NavListCore)`
  float: right;
  font-family: 'AvenirNextBold', 'Helvetica', 'sans-serif';
`

export default NavList;