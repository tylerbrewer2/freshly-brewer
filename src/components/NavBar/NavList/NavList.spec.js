import React from 'react';
import renderer from 'react-test-renderer';

import NavList from './NavList';

it('renders correctly', () => {
  const tree = renderer.create(
    <NavList>
      Test children
    </NavList>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
