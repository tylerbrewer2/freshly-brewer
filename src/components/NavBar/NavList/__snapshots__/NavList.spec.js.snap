exports[`test renders correctly 1`] = `
<ul
  className="sc-bdVaJa jnOPQH">
  Test children
</ul>
`;
